### ISSUE TITLE
Versions beginning with alphabet characters not recognized

### ISSUE THREAD

**Issue by mdmower** [download-monitor/download-monitor#263]
If I specify a version that begins with alphabet characters, then the version is not recognized when inserted into a download shortcode.

Steps to reproduce:

1. Create a download with two files
2. Version the files something like `fast-1.0` and `fast-1.1`
3. Insert the downloads into a post with tags:
```
    [download id="1259" version="fast-1.0"]
    [download id="1259" version="fast-1.1"]
```
Then, notice that the version is not recognized when viewing the post (the anchor does not include `?version="..."`). I can, however, manually paste `(url)/download/1259/?version=fast-1.0` into the URL bar to obtain the correct version.

WordPress 4.1.1
Download Monitor 1.7.0

**Comment by mdmower** [comment]
This should be fixed with [Pull Request 277](https://github.com/download-monitor/download-monitor/pull/277)

**Comment by barrykooij** [comment]
Merged #277 
Thanks @mdmower
